src/main/java/com/epam/aidial/cfg/dao/audit/repository/AdapterHistoryRepository.java

```diff
@@ -17,11 +19,15 @@
 @LogExecution
 public class AdapterHistoryRepository extends RevisionRepository {
 
+    private final ModelJpaRepository modelJpaRepository;
     private final AdapterJpaRepository adapterJpaRepository;
     private final AdapterEntityMapper adapterEntityMapper;
 
-
     public void rollbackAdapters(Number revision, AuditReader auditReader) {
+        Iterable<ModelEntity> models = modelJpaRepository.findAll();
```

Comment:
looks like independant part of PR. not related to PR title. or I miss something?

Reply:
Since this PR removes strict dependency between Model and Adapter, changes in AdapterHistoryRepository & ModelHistoryRepository are needed. Refactored the logic in them to match with Interceptor-InterceptorRunner & Application-ApplicationRunner